modules/Kconfig.st

```diff
@@ -3,10 +3,10 @@
 # Copyright (c) 2017 STMicroelectronics
 
 config HAS_STLIB
-	bool
+	bool "ST Micro time-of-flight library"
 
 config HAS_STMEMSC
-	bool
+	bool "ST Micro STdC (STMEMSC) library"
```

Comment:
This should not be selectable (revert)

Reply:
But I need to enable HAS_STMEMSC. I saw for example that you do ```select HAS_NORDIC_DRIVERS``` even if seems not selectable. I'm a bit confused here.